Validation/RecoTrack/python/TrackingParticleSelectionsForEfficiency_cff.py

```diff
@@ -12,7 +12,7 @@
     ptMin = cms.double(0.9),
     ptMax = cms.double(1e100),
     maxRapidity = cms.double(2.5),
-    tip = cms.double(3.5),
+    tip = cms.double(2.0),
```

Comment:
is this the reason for `DQMHistoTests: Total failures: 88444` in the last set of tests?

Reply:
That seems likely.

I'd leave it to the Tracking POG to decide if they want his change or not; if not, then I can drop it.

@mtosi @AdrianoDee

Reply:
Sincerely I'd revert it to 3.5 to take into account tips after barrel 1. @mtosi ?

Though, I don't remember/know why this was introduced.

Reply:
I'd guess the change was introduced before https://github.com/cms-sw/cmssw/pull/26700 as a "brute force" way to have the max tip below BPix layer 1 for phase1.